provision-base.sh

```diff
@@ -322,6 +350,13 @@ monthly_show_badconfig="YES"
 EO_PERIODIC
 }
 
+install_vimrc()
+{
+
+    tell_status "installing a vimrc"
+    curl https://raw.githubusercontent.com/wklken/vim-for-server/master/vimrc > "$BASE_MNT/root/.vimrc"
```

Comment:
1. this is likely to fail as `curl` has not been installed.
2. I'd rather see this included as a HEREDOC.